package.json

```diff
@@ -37,7 +37,7 @@
     ]
   },
   "peerDependencies": {
-    "webpack": "^4.0.0"
+    "webpack": "^5.0.0"
```

Comment:
shouldn't this be `^4.0.0 || ^5.0.0` as the plugin still support webpack 4 (as you did not change anything) ?